clients/node/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@ministryofjustice/hmpps-non-associations-api",
+  "version": "0.0.0",
+  "private": false,
+  "description": "HMPPS Non-associations API REST client",
+  "author": {
+    "name": "HMPPS Digital (MoJ Digital & Technology)",
+    "email": "info@digital.justice.gov.uk",
+    "url": "https://mojdigital.blog.gov.uk/"
+  },
+  "homepage": "https://github.com/ministryofjustice/hmpps-non-associations-api",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ministryofjustice/hmpps-non-associations-api.git"
+  },
+  "license": "MIT",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "/dist"
+  ],
+  "scripts": {
+    "build": "tsc",
+    "clean": "rm -rf ./dist/",
+    "format": "prettier --write src test",
+    "lint": "eslint . --cache --max-warnings 0",
+    "test": "jest"
+  },
+  "engines": {
+    "node": "18 || 20"
+  },
+  "dependencies": {
+    "agentkeepalive": ">=4",
+    "superagent": ">=8"
```

Comment:
direct dependencies are just minimal versions and should be set precisely in clients of this library